awx/main/utils/formatters.py

```diff
@@ -127,14 +127,14 @@ def reformat_data_for_log(self, raw_data, kind=None):
                 pass  # best effort here, if it's not valid JSON, then meh
             return raw_data
         elif kind == 'system_tracking':
-            data = copy(raw_data['ansible_facts'])
```

Comment:
a bunch of these keys are assumed to exist when using these loggers (which isn't the case for the test message)